tracer/test/snapshots/AspNetMvc4Tests.Classic.WithFF.__path=_Admin_Home_Index_statusCode=200.verified.txt

```diff
@@ -7,6 +7,28 @@
     Service: sample,
     Type: web,
     ParentId: Id_3,
+    Tags: {
+      aspnet.action: hellofromchild,
```

Comment:
It won't let me comment, but the resource for this looks wrong? Shouldn't it be `GET /admin/home/hellofromchild`? 🤔 (Looks wrong for all of the "with FF" ones)

Reply:
So I may be missing something on how this is supposed to work. But I think that the first issue is that we have [interverted NoFF and WithFF](https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetMvc4Tests.cs#L72). WDYT?

Reply:
> I think that the first issue is that we have interverted NoFF and WithFF. 

Yep, definitely looks that way! In which case the issue is with the no FF ones 😉 Which makes more sense - you're currently checking whether to replace `httpContext.Items[SharedItems.HttpContextPropagatedResourceNameKey` when `newResourceNamesEnabled`, but I don't know that you're doing the check otherwise (haven't dug through to find where this is set yet)

Reply:
Fixed in 2b0498b7dd359f9d9654a8ca09a540398c91d48